core/internal/cltest/cltest.go

```diff
@@ -266,6 +266,28 @@ func NewConfigWithWSServer(t testing.TB, url string, wsserver *httptest.Server)
 	return config
 }
 
+type JobPipelineV2TestHelper struct {
```

Comment:
We should probably use this pattern more in our tests

Reply:
Agreed. I think it'll just be a slow migration